app.py

```diff
@@ -138,6 +137,15 @@ def unsubscribe():
     return redirect(url_for("home", error_description="Sorry to see you go. You have been unsubscribed"))
 
 
+@app.route(f"{URL_PREFIX}/email_unsubscribe/<newsletter_id>")
```

Comment:
I'm wondering if a merge went weird or something? Because I deleted this last week I think.

Reply:
Yes, the merge went weird on my side. My local branch did not get updated with the latest changes in main branch, even after both `git merge` and `git rebase`. Latest, I did `reset --hard` to update my local branch and made changes again, and `push --force`.

Reply:
Yeah, **something** must have been funky, since using outdated code normally wouldn't give us this error. but it's fine. as long as we fixed it we're good.